compiler-tests/pom.xml

```diff
@@ -174,7 +182,23 @@
           </includedWasts>
           <!-- Skip Binary as there is a THROW_REF in a Global initializer -->
           <!-- Skip stack-overflows in tail calls -->
-          <excludedTests>SpecV1BinaryTest.test40,SpecV1TcReturnCallTest.test19,SpecV1TcReturnCallTest.test24,SpecV1TcReturnCallTest.test25,SpecV1TcReturnCallTest.test30,SpecV1TcReturnCallTest.test31,SpecV1TcReturnCallIndirectTest.test40,SpecV1TcReturnCallIndirectTest.test41,SpecV1TcReturnCallIndirectTest.test46,SpecV1TcReturnCallIndirectTest.test47</excludedTests>
+          <excludedTests>
+            SpecV1BinaryTest.test40,
+            SpecV1TcReturnCallTest.test19,
+            SpecV1TcReturnCallTest.test24,
+            SpecV1TcReturnCallTest.test25,
+            SpecV1TcReturnCallTest.test30,
+            SpecV1TcReturnCallTest.test31,
+            SpecV1TcReturnCallIndirectTest.test40,
+            SpecV1TcReturnCallIndirectTest.test41,
+            SpecV1TcReturnCallIndirectTest.test46,
+            SpecV1TcReturnCallIndirectTest.test47,
+            <!-- Skip exception tests with tail calls -->
+            SpecV1EhTryTableTest.test39,
+            SpecV1EhTryTableTest.test40,
+            <!-- Investigate why this one fails -->
+            SpecV1EhBinaryTest.test40
```

Comment:
@andreaTP I need an expert at reading binary WASM to explain to me what this test is doing..
So I assume this should pass the Parser, and it's up to the compiler to error it out.  Just not sure what I need to look out for.

```
;; Init expression with missing end marker
(assert_malformed
  (module binary
    "\00asm" "\01\00\00\00"
    "\01\04\01\60\00\00"       ;; Type section: 1 type
    "\03\02\01\00"             ;; Function section: 1 function
    "\06\05\01\7f\00\41\00"    ;; Global section: 1 entry with missing end marker
    ;; Missing end marker here
    "\0a\04\01\02\00\0b"       ;; Code section: 1 function
  )
  "unexpected end of section or function"
)
```

Reply:
IIRC the Global initializer contains > 1 instructions.

I left a comment here:
https://github.com/dylibso/chicory/blob/df5be87550b2d2347f0fc2e4ca5cb4c06a9751bc/runtime-tests/pom.xml#L182-L183

This test is supposed to fail differently depending on the enabled proposals, I think we can hold off a bit on being this strict, i.e. as soon as the module fails it's good enough.

I agree on skipping this one.